python/py-magic/Portfile

```diff
@@ -8,11 +8,10 @@ python.rootname     python-magic
 version             0.4.24
 revision            0
 categories-append   devel
-platforms           darwin
 license             MIT
 supported_archs     noarch
 
-python.versions     27 35 36 37 38 39 310
+python.versions     27 37 38 39 310
```

Comment:
Is there any reason to keep py27-magic subport?

Reply:
```
% port echo depends:py27-magic
fdroidserver
alot
```